docs/API.md

```diff
@@ -477,7 +477,7 @@ minioClient.fGetObject('mybucket', 'photo.jpg', '/tmp/photo.jpg', function(err)
 
 ```
 <a name="putObject"></a>
-### putObject(bucketName, objectName, stream, size, contentType, callback)
+### putObject(bucketName, objectName, stream, size, callback)
```

Comment:
Should I put it back in the documentation? Right now the parameter is still *supported*, it's just legacy optional — the new `ObjectUploader` doesn't require it. Or should I change something under the hood?